build_tools/github_actions/configure_ci_test.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import unittest
+
+import configure_ci
+
+SORTED_DEFAULT_BENCHMARK_PRESETS = ",".join([
+    "android-cpu",
+    "android-gpu",
+    "comp-stats",
+    "cuda",
+    "vulkan-nvidia",
+    "x86_64",
+])
```

Comment:
1. This will need to be kept in sync with the source file. Can you use that variable directly instead?
2. Rather than manually sort, how about calling .sort() in code? Or, even better - do a set comparison instead of a sorted list comparison (https://stackoverflow.com/questions/12813633/how-to-assert-two-list-contain-the-same-elements-in-python)
    * As part of that, you may want to make `get_benchmark_presets()` return a list, rather than have it join into a comma-separated string prematurely (only format structured data into a string at the specific location where that formatting is required, so recovering the structure isn't necessary)